invenio_query_parser/contrib/spires/config.py

```diff
@@ -1,25 +1,27 @@
+
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio-Query-Parser.
-# Copyright (C) 2014 CERN.
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
 #
-# Invenio-Query-Parser is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License as
+# INSPIRE is free software; you can redistribute it
```

Comment:
I think the copyright should still stay to `Invenio-Query-Parser`. In this way we can send upstream these changes.